test/evergreen.yml

```diff
@@ -2916,14 +2948,23 @@ buildvariants:
       PATH=/opt/mongodbtoolchain/v3/bin:$PATH
       CFLAGS="-fsanitize=address -fno-omit-frame-pointer -ggdb"
       CXXFLAGS="-fsanitize=address -fno-omit-frame-pointer -ggdb"
-    posix_configure_flags: --enable-silent-rules --enable-strict --enable-diagnostic --disable-static --prefix=$(pwd)/LOCAL_INSTALL
+    posix_configure_flags:
```

Comment:
Did we want to enable tcmalloc here?

Reply:
Ideally we should be enabling tcmalloc here, but we get multiple ASAN leak errors when we enable tcmalloc. More information about the ASAN leaks are described in WT-8194.